stuff/cron/update_ranks.py

```diff
@@ -118,7 +118,8 @@ def update_user_rank(cur: MySQLdb.cursors.BaseCursor) -> Sequence[float]:
         ) AS up
         INNER JOIN
             `Problems` AS `p`
-        ON `p`.`problem_id` = up.`problem_id` AND `p`.visibility > 0
+        ON `p`.`problem_id` = up.`problem_id` AND `p`.visibility > 0 AND
+        `p`.quality_seal = 1
```

Comment:
nit:
```suggestion
        ON
            `p`.`problem_id` = up.`problem_id`
            AND `p`.visibility > 0
            AND `p`.quality_seal = 1
```
para respetar la indentación